src/biz/workspace/ops.rs

```diff
@@ -53,6 +55,10 @@ pub async fn delete_workspace_for_user(
 
   // remove from postgres
   delete_from_workspace(&pg_pool, &workspace_id).await?;
+  let _: redis::Value = connection_manager
+    .del(UpdateStreamMessage::stream_key(&workspace_id))
+    .await
+    .map_err(|err| AppResponseError::new(ErrorCode::Internal, err.to_string()))?;
```

Comment:
how is the client handle this error? Maybe we just need to log it on the server side.

Reply:
Tbh. I think we could just retry here. I don't expect it to fail for any other reason than network issues.

Reply:
I've checked the client side implementation.  it should be fine